packages/bundle/src/adaptiveCards/Attachment/AdaptiveCardRenderer.js

```diff
@@ -84,19 +84,19 @@ function addEventListenerOnceWithUndo(element, name, handler) {
   return detach;
 }
 
+function addEventListenerWithUndo(element, name, handler) {
+  element.addEventListener(name, handler);
+
+  return () => element.removeEventListener(name, handler);
+}
+
 function disableElementWithUndo(element) {
   const undoStack = [];
```

Comment:
just wondering why do we call it **undo**stack?

Reply:
not sure - would have to ask compulim